maps/biter_battles_v2/functions.lua

```diff
@@ -454,6 +409,193 @@ function Public.map_intro_click(player, element)
+				if v == "gray" or v == "grey" then v = "military-science-pack" end
+				if v == "blue" then v = "chemical-science-pack" end
+				if v == "purple" then v = "production-science-pack" end
+				if v == "yellow" then v = "utility-science-pack" end
+				if v == "white" then v = "space-science-pack" end
+				local values = Tables.food_values[v]
+				if values == nil then
+					error_msg = "Invalid science pack color"
+				else
+					flask_color = v
+				end
+			elseif k == "count" then
+				if flask_color == nil then
+					error_msg = "Must specify flask color before count"
+				else
+					flask_count = tonumber(v)
+					if flask_count == nil or flask_count <= 0 or flask_count > 1000000000 then
+						error_msg = "Invalid flask count"
+					end
+					if foods[flask_color] == nil then foods[flask_color] = 0 end
+					foods[flask_color] = foods[flask_color] + flask_count
+				end
+				flask_count = tonumber(v)
+				if flask_count == nil or flask_count <= 0 or flask_count > 1000000000 then
+					error_msg = "Invalid flask count"
+				end
+				flask_color = nil
+			else
+				error_msg = string.format("Invalid parameter: %q", k)
+			end
+		else
+			error_msg = string.format("Invalid parameter: %q, must do things like \"evo=120\"", param)
+		end
+		if error_msg then break end
+	end
+	if flask_color ~= nil then
+		error_msg = "Must specify \"count\" after \"color\""
+	end
+	if error_msg == nil and #foods == 0 and player ~= nil then
+		local i = player.get_main_inventory()
+		for food_type, _ in pairs(Tables.food_values) do
+			local flask_amount = i.get_item_count(food_type)
+			if flask_amount > 0 then
+				foods[food_type] = flask_amount
+			end
+		end
+	end
+	if evo == nil and force_to_send_to then
+		local biter_force_name = force_to_send_to .. "_biters"
+		evo = global_passed_in.bb_evolution[biter_force_name] * 100
+	end
+	if error_msg == nil and evo == nil then
+		error_msg = "Must specify evo (or force)"
+	end
+	if error_msg then
+		return error_msg .. help_text
+	end
+	local total_food = 0
+	local debug_command_str = string.format("evo=%.1f difficulty=%d players=%d", evo,
+										    math.floor(difficulty), player_count)
+	for k, v in pairs(foods) do
+		total_food = total_food + v * Tables.food_values[k].value
+		debug_command_str = debug_command_str .. string.format(" color=%s count=%d", k, v)
+	end
+	if total_food == 0 then
+		error_msg = "no \"color\"/\"count\" specified and nothing found in inventory"
+	end
+	if error_msg then
+		return error_msg .. help_text
+	end
+	local effects = Public.calc_feed_effects(evo / 100, total_food * difficulty / 100, 1,
+										     player_count, global_passed_in.max_reanim_thresh)
+	return string.format("/calc-send %s\nevo_increase: %.1f new_evo: %.1f\nthreat_increase: %d",
+						 debug_command_str, effects.evo_increase * 100, evo + effects.evo_increase * 100,
+						 math.floor(effects.threat_increase))
+end
+
 function get_ammo_modifier(ammo_category)
```

Comment:
```suggestion
local function get_ammo_modifier(ammo_category)
```

not your code but we should local these